integrations/logging/loki/install.sh

```diff
@@ -71,7 +85,7 @@ kubectl get secret -n ${NAMESPACE} loki-basic-auth > /dev/null 2>&1
 
 if [ $? -ne 0 ]; then
   echo "Secret 'loki-basic-auth' not found, creating it..."
-  kubectl create secret -n ${NAMESPACE} generic loki-basic-auth --from-literal=username=admin --from-literal=password=${LOKI_PWD}
+  kubectl create secret -n ${NAMESPACE} generic loki-basic-auth --from-literal=username=admin=${LOKI_USERNAME} --from-literal=password=${LOKI_PWD}
```

Comment:
I think there is a typo here, maybe you wanted to write this?

```suggestion
  kubectl create secret -n ${NAMESPACE} generic loki-basic-auth --from-literal=username=${LOKI_USERNAME} --from-literal=password=${LOKI_PWD}
```
```suggestion
  kubectl create secret -n ${NAMESPACE} generic loki-basic-auth --from-literal=username=admin=${LOKI_USERNAME} --from-literal=password=${LOKI_PWD}
```
```suggestion
  kubectl create secret -n ${NAMESPACE} generic loki-basic-auth --from-literal=username=admin=${LOKI_USERNAME} --from-literal=password=${LOKI_PWD}
```